http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/monitor/MonitorAtBoot.scala

```diff
@@ -1,92 +1,191 @@
 package org.broadinstitute.dsde.workbench.leonardo
 package monitor
 
-import cats.effect.{Async, Timer}
+import cats.effect.{ConcurrentEffect, Timer}
```

Comment:
All changes in this file are rob's, so I'll let him respond to any feedback here and implement as needed since I'm not too familiar with this code. Note that as of submitting this PR, it is not heavily tested. (Unit tests are TODO)